crates/bevy_pbr/src/alpha.rs

```diff
@@ -2,16 +2,24 @@ use bevy_ecs::{component::Component, reflect::ReflectComponent};
 use bevy_reflect::std_traits::ReflectDefault;
 use bevy_reflect::Reflect;
 
-// FIXME: This should probably be part of bevy_render2!
-/// Alpha mode
+// TODO: add discussion about performance.
+/// Sets how a PBR material's base color alpha channel is used for transparency.
 #[derive(Component, Debug, Default, Reflect, Copy, Clone, PartialEq)]
 #[reflect(Component, Default)]
 pub enum AlphaMode {
+    /// Base color alpha values are overridden to be fully opaque (1.0).
     #[default]
     Opaque,
-    /// An alpha cutoff must be supplied where alpha values >= the cutoff
-    /// will be fully opaque and < will be fully transparent
+    /// Reduce transparency to fully opaque or fully transparent
+    /// based on a threshold.
+    ///
+    /// The base color texture pixels with an alpha channel lower than the
+    /// provided value will be fully transparent while pixels with an alpha
+    /// greater than the provided value will be fully opaque.
```

Comment:
I'm not sure I feel the first bit is clearer than it was.

```suggestion
    /// Compare the base color alpha value to a specified threshold. If the value is below the
    /// threshold, consider the color to be fully transparent (alpha is set to 0.0). If it is equal
    /// to or above the threshold, consider the color to be fully opaque (alpha is set to 1.0).
```

Reply:
I think the first sentence should be a short sentence summarizing the whole doc.

On a second read, I totally agree that doc comment is very confusing, and your proposed suggestion is a clear improvement.

I don't think "Compare the base color alpha value to a specified threshold" is an improvement over "Reduce transparency to fully opaque or fully transparent based on a threshold."

 I'd rather keep the first sentence and replace the rest of the doc with your suggested change. What do you think?